data/data_set.go

```diff
@@ -313,7 +313,7 @@ type DataSet struct {
 	Payload             *metadata.Metadata      `json:"payload,omitempty" bson:"payload,omitempty"`
 	Source              *string                 `json:"source,omitempty" bson:"source,omitempty"`
 	State               *string                 `json:"-" bson:"_state,omitempty"`
-	Time                *string                 `json:"time,omitempty" bson:"time,omitempty"`
+	Time                *string              `json:"time,omitempty" bson:"time,omitempty"`
```

Comment:
SuperTinyNit: is your editor changing spaces to tabs? :)

Reply:
we use tabs in the code, so i set my editor to tabs recently... usually have tabs set to 4 spaces.